recipes/nb2plots/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "nb2plots" %}
+{% set version = "0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ed3e62fad6701c0d3f4575b8a7037e69584e2d6e6cff47f786dd7a8c73fc9a6a
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - ipython[notebook] >=3.0
```

Comment:
Does this work?

Reply:
Probably not. @CurtLH did conda-skeleton added that or you did it manually?

Reply:
I used conda-skeleton to start the recipe, and then modified it from there.  But `ipython[notebook] >=3.0` came from conda-skeleton.  

It is in the [requirements.txt](https://github.com/matthew-brett/nb2plots/blob/6d00a39ab29fb62d024a4456a772379ff71546b8/requirements.txt#L3)

Let me know and I can open a PR to remove it from the feedstock if needed.